WpfExamples references to a project with net452 when itself is netcoreapp3.1

### Steps to reproduce

1. Call 
```cmd
dotnet build .\Source\Examples\WPF\WpfExamples\WpfExamples.csproj
```
2. See following
`C:\Program Files\dotnet\sdk\3.1.201\Microsoft.Common.CurrentVersion.targets(1655,5): warning NU1702: ProjectReference '.\oxyplot\Source\OxyPlot.Pdf\OxyPlot.Pdf.csproj' was resolved using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v3.1'. This project may not be fully compatible with your project. [.\oxyplot\Source\Examples\WPF\WpfExamples\WpfExamples.csproj]`

### Expected behaviour

Build without warning NU1702

### Actual behaviour

Build with warning NU1702

--- a/Source/Examples/WPF/WpfExamples/WpfExamples.csproj
+++ b/Source/Examples/WPF/WpfExamples/WpfExamples.csproj
@@ -6,7 +6,6 @@
       <OutputType>WinExe</OutputType>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\OxyPlot.Pdf\OxyPlot.Pdf.csproj" />
     <ProjectReference Include="..\..\..\OxyPlot.Wpf\OxyPlot.Wpf.csproj" />
     <ProjectReference Include="..\..\..\OxyPlot\OxyPlot.csproj" />
   </ItemGroup>

